### PR TITLE
Add SIG CLI Spotlight blog article

### PR DESCRIPTION
**Pull request by fsmunoz**
SIG CLI Spotlight article.

### CODE REVIEW

**Comment by fsmunoz** [comment]
Please review and confirm, especially because while editing I've added some small changes like the links to the GitHub profiles, maybe you prefer to link to Twitter or somewhere else.

/cc @KnVerey @soltysh

**Comment by arpitjindal97** [comment]
You guys tagged wrong Arpit 😆

**Comment by fsmunoz** [comment]
@arpitjindal97 apologies, will correct the commit, thanks for the heads-up!

**Comment by sftim** [comment]
It would be nice to have a PR against k/website ready so that we mirror this article there (with this one as canonical)

See other articles in this series for hints about how we've done that.

**Comment by sftim** [comment]
/retitle Add SIG CLI Spotlight blog article

File: content/en/blog/2023/sig-cli-spotlight.md [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+are irreplaceable.
+
+**Arpit**: Kubernetes is a community-driven project. Any
+recommendation for anyone looking into getting involved in SIG CLI
+work? Where should they start? Are there any prerequisites?
+
+**Maciej**: There are no prerequisites other than a little bit of free
+time on your hands and willingness to learn something new :-)
+
+**Katrina**: A working knowledge of Go often helps, but we also have
```
Comment (sftim) [inline content/en/blog/2023/sig-cli-spotlight.md:156]: (nit) we could link to https://go.dev/ here.

**Comment by fsmunoz** [comment]
@sftim sure, added https://github.com/kubernetes/website/pull/41973 for the website.

File: content/en/blog/2023/sig-cli-spotlight.md (hunk 1 above)
Comment (fsmunoz) [reply content/en/blog/2023/sig-cli-spotlight.md]: Done!

**Comment by fsmunoz** [comment]
Going to change the publishing date header if no updates during the day .

**Comment by sftim** [comment]
I expected that https://github.com/kubernetes/website/pull/41973/ would be filled in a lot more. I'd prefer to wait until both are ready to go.

/hold

How about bumping these both back 7 days, and filling in https://github.com/kubernetes/website/pull/41973/ so that it previews OK? Have a look at other SIG spotlight article PRs in k/website to see how this is done.

OK to unhold once the other PR is reviewable.

**Comment by fsmunoz** [comment]
Hi @sftim, thanks for the comments. I kept it empty on purpose, for two reasons:

* If we are doing the review here until it's ready, it would be a waste of time to replicate each change to the other one. Since I had no indication that this one was already completely reviewed, I didn't update the other one.
* In previous SIG Spotlight articles (like https://github.com/kubernetes/website/pull/35084 ), it was explicitly mentioned that it should first be approved here, and then "copied" to the other - which makes sense and avoids work, but my understanding of this was to push a placeholder there and wait for the document to be ready here before doing the "copying"

I can easily push a commit to the other one right now, with the existing content, and changing the publishing date as suggested. We should then better document the process (I can update the blogging guidelines we have in Comms).

**Comment by fsmunoz** [comment]
Changed date, rebased, updated https://github.com/kubernetes/website/pull/41973 with existing content, as discussed!

**Pull request merged**